.drone.yml

```diff
@@ -123,50 +123,6 @@ pipeline:
     when:
       status: success
 
-  vic-ui:
```

Comment:
Do we have corresponding change to include `vic-ui` artifacts in OVA build?

Reply:
We are no longer publishing vic-ui binaries as part of https://github.com/vmware/vic-product/pull/1742.
The changes to provision the vic-ui plugin zips archives are in https://github.com/vmware/vic-product/pull/1758.

Reply:
Cool, thanks!